src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -21,7 +21,9 @@ namespace System.Security.Cryptography.Pkcs.EnvelopedCmsTests.Tests
 {
     public static partial class DecryptTests
     {
-        [Fact]
+        public static bool IsNet462OrNewer() => (!PlatformDetection.IsFullFramework || PlatformDetection.IsNetfx462OrNewer());
```

Comment:
@bartonjs - what I'm saying is that `PlatformDetection.IsNetfx462OrNewer()` should always return false on netfx which implies that this logic should be moved to `PlatformDetection.IsNetfx462OrNewer()` directly